src/Microsoft.AspNetCore.Razor.Tasks/DotnetToolTask.cs

```diff
@@ -170,9 +170,8 @@ private string CurrentDirectoryToUse()
 
         private List<string> GetArguments(string responseFileCommands)
         {
-            var responseFileArguments =
-                CommandLineUtilities.SplitCommandLineIntoArguments(responseFileCommands, removeHashComments: true);
```

Comment:
We don't really need this because we don't use the `commandLineArguments` and we format `responseFileArguments`  ourselves and we don't add hash comments. I feel the below code is much simpler and less bug prone. Correct me if I am missing something.